### ISSUE TITLE
when DATA_SET == 'small ': dev_index_HGG, test_index_HGG is empty

### ISSUE THREAD

**Issue by sharmarochan** [zsdonghao/u-net-brain-tumor#37]
if we select the DATA_SET=='small' and  print (dev_index_HGG, test_index_HGG ), it  returns the blank arrays.

**Comment by guido-niku** [comment]
i encoutered this problem, actually they are not blank, i used randomized index to manually display some images, and i found that many were blank images, while others did have the proper pictures. Remember those are 2d images sliced from 3d volumes (if i am not mistaken) .  Not sure if this is to be expected or not ( maybe a slice of a volume that effectively contains nothing ?

**Comment by sharmarochan** [comment]
Yes you are right, the problem was that it cannot be downloaded without git lft. I downloaded the dataset with python script.
